docs/installing-northstar/northstar-installers/viper-guide.md

```diff
@@ -0,0 +1,75 @@
+
+![Installed mods list](../../images/viper-installed-mods.png)
+
+This menu (when opened) shows you a list of your currently enabled mods, with version numbers, author names, and toggles to disable or remove them individually.
+
+- `Remove All` would do just that, removing all of your mods.
+- `Toggle All` would also do just that, toggling all of your mods.
+- `Install Mod` is a button you can press and locate a mod folder installed to your computer, and Viper will install it for you.
+
+![Viper's Thunderstore browser](../../images/viper-find-mods.png)
+
+The `Find Mods` button on Viper's mod screen allows you to search through [Northstar's Thunderstore page](https://northstar.thunderstore.io/) inside of Viper with the ability to quickly download a mod, as well as seeing the Mod's author and version number.
+
+![Viper's release notes for Northstar](../../images/viper-northstar-release-notes.png)
+
+The `Release Notes` tab shows you the release notes for Northstar versions, which you can find [here](https://github.com/R2Northstar/Northstar/releases)
+
+### Viper Menu 
+
+![Viper's main menu](../../images/viper-menu.png)
+
+This menu of Viper (the top one on the left sidebar) shows you the version of Viper you have installed, and an option to set your game path. Viper _should_ do this automatically, but some users may need to manually set this. Default directories can be found [here](../troubleshooting.md#game-location), or if you own the game on EA and have followed the [EA App directory issue section](../troubleshooting.md#cannot-write-log-file-when-using-northstar-on-ea-app), another common location is `C:\Games\Titanfall2`
+
+![Viper's release notes](../../images/viper-release-notes.png)
+
+This page shows the release notes for Viper versions, which you can find [here](https://github.com/0neGal/viper/releases)
+
+The `Extras` tab of Viper displays some additional information, like credit for backgrounds used in Viper.
+
+### Vanilla Menu
+
+![Viper's Vanilla menu](../../images/viper-vanilla.png)
+
+This menu allows you to Launch vanilla Titanfall 2 (no mods) easily, by simply hitting `Launch` on this screen
+
+## Viper's Invalid Config issue
+
+![The Invalid Config error](../../images/viper-invalid-config.png)
+
+This error should be mostly resolved as of Viper version `1.7.2`, however if clicking `OK` doesn't seem to fix the error, you can manually delete the config file and allow Viper to recreate a functional one. You can do this on Windows by pressing `Windows + R`, typing in `%appdata%`, hitting `Enter`, then finding and deleting `viper.json` (if you can't see the `.json`, it's because Windows hides file extensions by default. You can turn them on by hitting `View` in task manager, then ticking the box that says `File name extensions`)\
+On Linux, `viper.json` can be found at either `$XDG_CONFIG_HOME/viper.json` or `$HOME/.config/viper.json` (if the XDG variable isn't set)
```

Comment:
@0neGal something like this good, or should I word it differently?

Reply:
Hmm, something like this might be better...

> On Linux, `viper.json` can be found in either the `$XDG_CONFIG_HOME` folder, if the variable is set, and if not it'll be in the `$HOME/.config` folder.

Reply:
done